casper/src/main/resources/AuthKey.rho

```diff
@@ -67,12 +71,31 @@ in {
       }
     }
   } |
+  // No internal state to return.
+  // Method required for updates.
+  for (ret <- @(*AuthKey, "extractState")) {
+    ret!({"authKeyUnf": *_authKey})
+  }  |
+
+  rl!(`rho:registry:systemContractManager`, *systemContractManagerCh)|
+  rl!(`rho:rchain:pubKeyCheck`, *pubKeyCheckCh)|
+  for (@(_, systemContractManager) <- systemContractManagerCh & @(_, pubKeyCheck)<- pubKeyCheckCh){
+    @systemContractManager!("createDispatcher", *AuthKey, *dispatcherCh)|
+    contract @(*AuthKey, "securityCheck")(@deployerId, ret) = {
+      @pubKeyCheck!(
+        deployerId,
+        "04e2eb6b06058d10b30856043c29076e2d2d7c374d2beedded6ecb8d1df585dfa583bd7949085ac6b0761497b0cfd056eb3d0db97efb3940b14c00fff4e53c85bf",
```

Comment:
This is going to be problem when we want to put this hard-coded public key in the configuration.

So instead to duplicate config public key in all files it makes sense to have it only in the Registry where we can substitute only once, similar how we have PoS contract (it not great solution but we don't have better right now).